packages/react-native/Libraries/Lists/FlatList.js

```diff
@@ -642,22 +638,24 @@ class FlatList<ItemT> extends React.PureComponent<Props<ItemT>, void> {
 
     const renderProp = (info: RenderItemProps<ItemT>) => {
       if (cols > 1) {
-        const {item, index} = info;
+        const { item, index } = info;
```

Comment:
This line should remain the same.
```suggestion
        const {item, index} = info;
```